docs/user_guide.rst

```diff
@@ -290,8 +290,52 @@ When executing a circuit that uses something other than the device qubits, you n
 as explained in the :ref:`routing` section above.
 
 
-More examples
--------------
+More advanced examples
+----------------------
+
+In this section we demonstrate some of the more advanced examples of using Cirq on IQM and how you can use various tools available in Cirq.
+
+Multiple circuits can be submitted to the IQM quantum computer at once using the :meth:`~.IQMSampler.run_iqm_batch` method of :class:`.IQMSampler`. 
+
+.. code-block:: python
+   
+   circuit_list = []
+
+   circuit_list.append(routed_circuit_1)
+   circuit_list.append(routed_circuit_2)
+
+   results = sampler.run_iqm_batch(circuit_list, repetitions=10)
+
+   for result in results:
+        print(result.histogram(key="m"))
```

Comment:
This would fit better under the section `Running on a real quantum computer` with a note on performance improvements when running circuits in a batch.

Reply:
Resolved in [76f994b](https://github.com/iqm-finland/cirq-on-iqm/pull/117/commits/76f994bc08e2c3f7e6a3351ac8cbb31d13ebc402)